### ISSUE TITLE
Call for Update "lastUpdatedTimestamp"

### ISSUE THREAD

**Issue by fboulegue** [bdr99/pymazda#10]
Is there a way to request a dataset update?
Right now I always get the same timestamp etc..., like I would need to ask the car for send a update

**Comment by bdr99** [comment]
Unfortunately I don't think there is a way to request the car to send an update. This feature would need to be in the official (MyMazda) app for me to add it to this library. If you find a way to request an update in the official app, please let me know :)

I've noticed that it seems to send an update whenever the engine is shut off, or when the doors are locked/unlocked. There may be other triggers as well, but I haven't found a way to manually request an update.

**Comment by fboulegue** [comment]
There is a way - If you go into Status and ‚pull to refresh‘ it Actually requesting new data 

![6D1A9FC7-B947-4520-918A-79916B708C1C](https://user-images.githubusercontent.com/2719590/120137824-716b8a00-c1d5-11eb-9253-ffaa744b6223.png)
Click on small arrow to fullpage

![4D0D3CE5-A6D1-44DF-BAEC-26945AD7B3AB](https://user-images.githubusercontent.com/2719590/120137843-7d574c00-c1d5-11eb-98c3-67f042ff7365.png)
Drag down like next picture 

![Uploading 0AE7831B-7939-4973-9D43-DDDE7E200F81.png…]()


Got it?

**Comment by bdr99** [comment]
Hmm, are you sure it is really requesting new data? When you drag down to refresh, does the timestamp at the top of the page change? With my car, the timestamp never changes, no matter how many times I drag down to refresh.

By the way, I can see your first two screenshots above, but I can't see the third one.

**Comment by fboulegue** [comment]
Yes I m sure ... it sometimes take a couple of sec until a minute but there is even a notification that give the information 'new real time data available' after the pull to refresh.

**Comment by fboulegue** [comment]
![image](https://user-images.githubusercontent.com/2719590/120256464-7b04f880-c28e-11eb-9b07-52de33ac7005.jpeg) see here it's the request push and after a few seconds there is a other one that's tells new information ready

**Comment by bdr99** [comment]
Interesting. I think this may be a unique feature for the MX-30. I'll do some research and see if I can figure out how to trigger this feature programmatically.

**Comment by fboulegue** [comment]
Sure let me know if you need any assist

**Comment by bdr99** [comment]
Hi @fboulegue 

I've been working on adding some MX-30 specific features. One of them is `refresh_vehicle_status()` which I believe will request an update from the vehicle. There are also other features like `get_ev_vehicle_status()`, `get_hvac_setting()`, `set_hvac_setting()`, `turn_on_hvac()`, and `turn_off_hvac()`.

I don't have an MX-30 for testing, so would you be able to test these features on your car? They are pushed to a branch named [`ev_hvac`](https://github.com/bdr99/pymazda/tree/ev_hvac). The new features are documented in the readme on that branch.

**Comment by bdr99** [comment]
Closing this issue since @CacioGit was able to test and confirm that `refresh_vehicle_status()` was working for them in this comment: https://github.com/bdr99/pymazda/issues/14#issuecomment-964282861. This feature is released in version 0.3.0.
